.github/actions/spelling/allow.txt

```diff
@@ -0,0 +1,325 @@
+IntelliJ
+noopener
+acmedtssdstest
+admins
+adoptopenjdk
+afds
+wdm
+portile
+afdverify
+aks
+deno
+dra
+alex
+allejo
+IaaS
+Iaa
+pipeable
+SaaS
+PaaS
+haml
+WCAG
+sassc
+Tful
+RESTful
+hashie
+prebuilds
+acme
+acmebot
+racc
+TGL
+nta
+toc
+aad
+AAD
+aat
+rb-inotify
+acr
+ada
+adf
+alphagov
+parslet
+apastuhov
+ccd
+apim
+apps
+artifactory
+Frontdoors
+atlassian
+servolux
+Vnets
+autoconfigure
+@U1234ABCD
+ABCD
+nsenter
+autoscaler
+azuread
+tfvars
+azurecr
+azuredevops
+azureedge
+azurefd
+paa
+Intelli
+azurerm
+backports
+barrelsby
+basarat
+benlesh
+betterfm
+bfaac
+bitnami
+blobstorage
+blockquote
+brunton
+cdk
+cdn
+cdnverify
+certficates
+certs
+cfb
+cft
+cftapps
+cftprod
+cftptl
+checkstyle
+CJS
+cleardown
+cloudapp
+cnp
+codeburst
+codeowners
+codinghorror
+commonmarker
+contino
+continuousdelivery
+cppdev
+crictl
+customise
+cyclomatic
+datastore
+dba
+DBaa
+dburl
+dcd
+dce
+deserialise
+dts
+dtspo
+dynatrace
+envsubst
+erb
+erubis
+eventmachine
+excludesfile
+execjs
+expressjs
+fastimage
+ffi
+firebaseapp
+firefox
+firewalls
+fluxcd
+flywaydb
+fprofiles
+frontdoor
+frontends
+fsevent
+getbem
+gitbook
+gitops
+gitpod
+gitrepositories
+gkf
+googling
+gotchas
+gotk
+govuk
+gradle
+gradlew
+hackernoon
+helmrelease
+hmcts
+hmctsjobs
+hmctspublic
+hmi
+hotspot
+howto
+hpa
+idam
+imagepolicies
+imagepolicy
+imagerepositories
+img
+intsvc
+ithc
+itnext
+jattach
+javaguide
+jayne
+jcmd
+jdbc
+jdk
+jenkins
+jenkinsfile
+jfr
+jira
+jitpack
+jmc
+jpa
+jsonpath
+jvm
+keyvault
+keyvaults
+klaus
+klu
+kramdown
+krew
+kubectl
+kubectx
+kubernetes
+kustomization
+kustomizations
+kustomize
+kvaps
+letsencrypt
+lifecycle
+linting
+liskov
+lvc
+macrotask
+mafr
+malcoded
+manully
+maxsize
+memoist
+mgmt
+microservices
+microsoft
+microsoftonline
+microtask
+ministryofjustice
+mozilla
+mswin
+muchcoffee
+multipage
+myaccess
+myapps
+nationalarchives
+ncforminfo
+ncsc
+netbasal
+netxm
+nfrs
+ngmodules
+ngrx
+nikpoltoratsky
+nokogiri
+NONINFRINGEMENT
+npmjs
+onboarded
+onpush
+ooc
+osba
+owasp
+palantir
+palo
+paypal
+pcq
+perftest
+pid
+platops
+pls
+POCs
+postgresql
+prebuild
+prioritise
+Protips
+ptl
+ptlsbox
+QAs
+Rawstorne
+recordsets
+redcarpet
+refactorings
+retagging
+rexml
+rpe
+rse
+rubygems
+runbooks
+rvm
+rxmarbles
+sbox
+sdshmcts
+sdsprod
+SDSPTL
+servicebus
+sharedservices
+SHAREDSERVICESPTL
+shibayan
+showpad
+shuttername
+Shuttersolution
+sigs
+simov
+slackernoon
+SLAs
+slf
+SOGs
+sonarcloud
+sonarqube
+spective
+spellcheck
+springframework
+sscs
+sscscor
+sso
+stackoverflow
+stakeholders
+Staticfile
+statuspage
+styleguide
+subdomain
+switchmap
+tabindex
+tablist
+tabpanel
+templated
+templating
+terraform
+terraformversion
+Teyk
+tfvar
+TLS
+TNJ
+toml
+toptal
+Tqc
+TRO
+trunkbaseddevelopment
```

Comment:
this is in a link I'm surprised it complains about this.

Reply:
This was left over in the allow list from before adding https regex 😬 Will try to spot anymore too